### ISSUE TITLE
request: publish docker images

### ISSUE THREAD

**Issue by joemiller** [ycd/toc#8]
It would be helpful if `toc` was available as a docker image. I think a simple scratch container with a static binary in it would be sufficient.

I'd offer a PR to your goreleaser.yaml but I can't really test it and don't know which docker registry you would prefer.

**Comment by ycd** [comment]
Hey @joemiller, just added it!

It is available with [v0.2.51](https://github.com/ycd/toc/releases/tag/v0.2.51).

```
docker run --rm -it yagizcan/toc:latest toc
docker run --rm -it yagizcan/toc:$TAG_NAME toc
```

**Comment by joemiller** [comment]
awesome!! thanks
